phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -1780,15 +1780,15 @@ private  static void verifySequence(String tenantID, String sequenceName, String
     
 
     /**
-     *  Split SYSTEM.CATALOG at the given split point 
+     *  Split a table at the given split point
      */
-    protected static void splitRegion(byte[] splitPoint) throws SQLException, IOException, InterruptedException {
+    protected static void splitRegion(TableName tableName, byte[] splitPoint) throws SQLException, IOException, InterruptedException {
         Admin admin =
                 driver.getConnectionQueryServices(getUrl(), TestUtil.TEST_PROPERTIES).getAdmin();
-        admin.split(PhoenixDatabaseMetaData.SYSTEM_CATALOG_HBASE_TABLE_NAME, splitPoint);
+        admin.split(tableName, splitPoint);
         // make sure the split finishes (there's no synchronous splitting before HBase 2.x)
-        admin.disableTable(PhoenixDatabaseMetaData.SYSTEM_CATALOG_HBASE_TABLE_NAME);
-        admin.enableTable(PhoenixDatabaseMetaData.SYSTEM_CATALOG_HBASE_TABLE_NAME);
+        admin.disableTable(tableName);
```

Comment:
Why do we need this? 
Regions go offline anyways when they split and `SplitRegionRequest` is coming from `BlockingInterface` in HBase 1.x versions.
For 2.x and above you can use `splitRegionSync` method

Reply:
So that the split completes, or else we have to keep checking the number of regions.

Reply:
The call to split() method should get blocked until the split is complete, isn't it?

Reply:
No its async in 1.x

Reply:
I am not sure if I am missing something over here, `BlockingService` proto service is being used for `AdminProtos` in HBase 1.3 or 1.4, which is the same being used for `MetaDataProtos` as well.

Reply:
That ends up calling CompactSplitThread.requestSplit which just enqueues the task.